packages/fxa-auth-server/lib/routes/subscriptions/paypal.ts

```diff
@@ -49,8 +57,20 @@ export const paypalRoutes = (
     stripeHelper
   );
 
-  // FIXME: All of these need to be wrapped in Stripe error handling
-  // FIXME: Many of these stripe calls need retries with careful thought about
-  //        overall request deadline. Stripe retries must include a idempotency_key.
-  return [];
+  return [
+    {
+      method: 'POST',
+      path: '/oauth/subscriptions/paypal-checkout',
+      options: {
+        auth: {
+          payload: false,
+          strategy: 'oauthToken',
+        },
+        response: {
+          schema: isA.string(),
```

Comment:
Our endpoints return JSON, so I'd expect the schema here to be an object with a token property.